### PR TITLE
{tools}[foss/2023a] VASPKIT v1.5.1

### PR DESCRIPTION
**Pull request by WilleBell**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by WilleBell** [comment]
Test report by @WilleBell
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node605.hydra.os - Linux Rocky Linux 8.9, x86_64, AMD EPYC 9384X 32-Core Processor, Python 3.6.8
See https://gist.github.com/WilleBell/26cd34a6e6b1974202f77fbd045916f7 for a full test report.

File: easybuild/easyconfigs/v/VASPKIT/VASPKIT-1.5.1_fix-envvars.patch [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+ GW_POTCAR                     =     .FALSE.                        # .TRUE. or .FALSE.; For example, H_GW, O_GW will be chose when POTCAR_GW set to .TRUE.
+ RECOMMENDED_POTCAR            =     .TRUE.                         # .TRUE. or .FALSE.; The recommended PAW potential will be chose when RECOMMENDED_POTCAR set to .TRUE.
+@@ -42,7 +44,7 @@
+ INTERPOLATION_SPACING         =      0.04                          # Determines the number of interpolation grids, in unit of A in real-space or 1/A in reciprocal space (default: 0.04)
+ INTERPOLATION_METHOD          =     'cubic'                        # 'linear', 'cubic' (3rd order-spline interpolation), quartic (4th order-spline interpolation), or FFT available only for 2D and 3D grids (default method: 'cubic')
+ AUTO_SUBMIT_JOB               =     .FALSE.                        # .TRUE. or .FALSE. (default: .FASLE.). Whether to auto-submit vaspkit or vasp job or not.   
+-SUBMIT_JOB_COMMAND            =     'qsub job.sh'                  # The command line to submit job
++SUBMIT_JOB_COMMAND            =     ''                             # The command line to submit job
```
Comment (smoors) [inline easybuild/easyconfigs/v/VASPKIT/VASPKIT-1.5.1_fix-envvars.patch:42]:
```suggestion
+SUBMIT_JOB_COMMAND            =     'sbatch job.sh'                # The command line to submit job
```

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen3

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @WilleBell!

**Pull request merged**